nixos/modules/services/mail/roundcube.nix

```diff
@@ -148,21 +153,20 @@ in
         ${cfg.hostName} = {
           forceSSL = mkDefault true;
           enableACME = mkDefault true;
+          root = cfg.package;
           locations."/" = {
-            root = cfg.package;
             index = "index.php";
             extraConfig = ''
-              location ~* \.php(/|$) {
-                fastcgi_split_path_info ^(.+\.php)(/.+)$;
-                fastcgi_pass unix:${fpm.socket};
-
-                fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
```

Comment:
The archwiki references this param as well. Why was it removed?

Reply:
This setting is already included in fastcgi.conf file.
```diff
--- /nix/store/56k504vw3yvjr4hv3x0fy4zv0gl6m888-nginxQuic-1.25.3/conf/fastcgi_params    1970-01-01 00:00:01.000000000 +0000
+++ /nix/store/56k504vw3yvjr4hv3x0fy4zv0gl6m888-nginxQuic-1.25.3/conf/fastcgi.conf      1970-01-01 00:00:01.000000000 +0000
@@ -1,4 +1,5 @@

+fastcgi_param  SCRIPT_FILENAME    $document_root$fastcgi_script_name;
 fastcgi_param  QUERY_STRING       $query_string;
 fastcgi_param  REQUEST_METHOD     $request_method;
 fastcgi_param  CONTENT_TYPE       $content_type;

```